tests/integration/integration_tests.py

```diff
@@ -376,8 +376,13 @@ async def main():
             logger.error(f"Invalid providers specified: {', '.join(invalid_providers)}")
             logger.error(f"Available providers: {', '.join(available_providers)}")
             sys.exit(1)
-    else:
-        selected_providers = available_providers
+        else:
+            selected_providers = available_providers
+
+    # Exclude ollama provider if it's in the list
+    if "ollama" in selected_providers:
+        selected_providers.remove("ollama")
+        logger.info("Excluded 'ollama' provider from tests.")
```

Comment:
I think we can leave that there as we might want to run the test locally. 

The change in the workflow yml file should be sufficient to skip the ollama tests from running 👍